src/config-import.js

```diff
@@ -183,6 +199,60 @@ export async function readConfig(configPath) {
               ? preset.icon
               : undefined,
           value: presetValue,
+          name,
+        }
+      }
+    },
+    /**
+     * @returns {Iterable<{name: string,value:import('@mapeo/schema').TranslationValue}>}
+     */
+    *translations() {
+      if (!translationsFile) return
+      for (let [lang, languageTranslations] of Object.entries(
+        translationsFile
+      )) {
+        const { language: languageCode, region: regionCode } = parseBCP47(lang)
+        if (!languageCode) throw new Error(`invalid translation language`)
+        for (let [
+          schemaNameRef,
+          languageTranslationsForDocType,
+        ] of Object.entries(languageTranslations)) {
+          if (!isRecord(languageTranslationsForDocType))
+            throw new Error(`invalid translation docType`)
+          for (let [docName, fieldsToTranslate] of Object.entries(
+            languageTranslationsForDocType
+          )) {
+            if (!isRecord(fieldsToTranslate))
+              throw new Error(`invalid translation field`)
+            for (let [fieldRef, message] of Object.entries(fieldsToTranslate)) {
+              // we skip messages that aren't strings, for now
+              if (isRecord(message)) continue
```

Comment:
When would `message` ever be a record?

Reply:
It shouldn't actually, it was a misconception I had based on badly parsing the file...

Reply:
PR updating translation options in fields -> https://github.com/digidem/mapeo-default-config/pull/58

Reply:
PR to update default-config -> https://github.com/digidem/mapeo-default-config/pull/58